util/template.go

```diff
@@ -0,0 +1,41 @@
+package util
+
+import (
+	"github.com/BurntSushi/toml"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+func FileToString(Filename string) string {
+	file, err := os.Open(Filename) // For read access.
+	if err != nil {
+		log.Fatal(5, "Error with configuration file: %s", Filename)
+	}
+	defer file.Close()
+	content, err := ioutil.ReadAll(file)
+	if err != nil {
+		log.Fatal(5, "Error with configuration file: %s", Filename)
+	}
+	str := string(content)
+	return str
+}
+
+type StoreConfigDatabase struct {
+	Type                  string `toml:"type"`
+	Create_DB_template    string `toml:"create_db_template"`
+	Create_Table_template string `toml:"create_table_template"`
```

Comment:
as I pointed out in my comment in #888 I don't think we should generalize that a store has 1 createDB and 1 createTable template, it depends on the store

Reply:
okay i'll make it specific